sys/sys/malloc.h

```diff
@@ -232,7 +232,7 @@ void	*malloc(size_t size, struct malloc_type *type, int flags) __malloc_like
  */
 #define	malloc(size, type, flags) ({					\
 	void *_malloc_item;						\
-	size_t _size = (size);						\
```

Comment:
Nit on the commit log for kernel malloc: s/the up size/the size up/

(Can't comment on the log message directly unfortunately)